pkg/data/management/role_data.go

```diff
@@ -220,10 +220,12 @@ func addRoles(wrangler *wrangler.Context, management *config.ManagementContext)
 		addRule().apiGroups("management.cattle.io").resources("clusterroletemplatebindings").verbs("get", "list", "watch")
 
 	rb.addRoleTemplate("Manage Cluster Catalogs", "clustercatalogs-manage", "cluster", false, false, true).
-		addRule().apiGroups("management.cattle.io").resources("clustercatalogs").verbs("*")
+		addRule().apiGroups("management.cattle.io").resources("clustercatalogs").verbs("*").
+		addRule().apiGroups("catalog.cattle.io").resources("clusterrepos").verbs("*")
```

Comment:
Yes, the `clusterrepos.catalog.cattle.io` CRD is installed in 2.6 line:
`kubectl get crds | grep clusterrepos` -> `clusterrepos.catalog.cattle.io`

Instead of replacing, we're adding it an additional policy rule. This allows the RoleTemplate to work for clustercatalogs (management.cattle.io) and for clusterrepos (catalog.cattle.io)